app/Console/Commands/SendScheduledMessages.php

```diff
@@ -51,9 +51,15 @@ public function handle(Sms $sms)
 
                 foreach ($subscribers as $subscriber) {
                     foreach ($scheduled_messages as $message) {
+                        $body = $message->body_en;
+                        switch ($subscriber->locale) {
+                        case 'es':
+                            $body = $message->body_es;
```

Comment:
The Twilio API will throw an exception if `body_es` is empty, but maybe we should fall back to the english version or `continue` instead of throw an exception?

Reply:
I was hoping to make the `body_es` non-nullable in the DB but ran into some issues and punted on it. However, the form validation ensures that a new scheduled message should always have a `body_es`.

Catching here as well is being more robust though, thanks for pointing that out!